components/inputs/inputs.css

```diff
@@ -0,0 +1,65 @@
+@import url(/css/base.css);
+@import url(/css/utility.css);
+
+/* input component start here */
+.et_input-container{
+align-items: center;
+display: flex;
+flex-direction: column;
+justify-content: center;
+}
```

Comment:
use prettier to increase readability

Reply:
> small changes, can be merged

hey ozair, thanks for the suggestions, will certainly do it! Thankyou so much for reviewing!!